conftest.py

```diff
@@ -123,4 +98,30 @@ def w3_strict_abi():
 
 @pytest.fixture(autouse=True)
 def print_warnings():
-    warnings.simplefilter('always')
+    warnings.simplefilter("always")
+
+
+# --- async --- #
+
+
+def is_async_testrpc_provider(provider):
+    return isinstance(provider, AsyncEthereumTesterProvider)
+
+
+@pytest.fixture()
+def async_skip_if_testrpc():
```

Comment:
similar question here, I thought we removed `skip_if_testrpc` in one of the previous filter PRs, do we need this still?